routers/api/v1/shared/action.go

```diff
@@ -152,6 +155,77 @@ func ListRuns(ctx *context.APIContext, ownerID, repoID int64) {
 		opts.CommitSHA = headSHA
 	}
 
+	// Handle exclude_pull_requests parameter
+	if exclude := ctx.FormString("exclude_pull_requests"); exclude != "" {
+		if exclude == "true" || exclude == "1" {
+			opts.ExcludePullRequests = true
+		}
+	}
+
+	// Handle check_suite_id parameter
+	if checkSuiteID := ctx.FormInt64("check_suite_id"); checkSuiteID > 0 {
+		opts.CheckSuiteID = checkSuiteID
+	}
+
+	// Handle created parameter for date filtering
+	if created := ctx.FormString("created"); created != "" {
+		// Parse the date range in the format like ">=2023-01-01", "<=2023-12-31", or "2023-01-01..2023-12-31"
+		if strings.Contains(created, "..\u002e") {
```

Comment:
`\u002e` is also dot?

Reply:
Addressed, see https://github.com/go-gitea/gitea/pull/34894#issuecomment-3016900408 and https://github.com/go-gitea/gitea/pull/34894/commits/8b7ed3af421ade2adabc9896bca8d4871029b2d0